it/mail/chrome/messenger/preferences/preferences.properties

```diff
@@ -1,49 +1,49 @@
 #### Add HTML and Plain Text Domain Names for sendOption
 
-html_domainsAddDomainTitle=Aggiungere nome dominio HTML:
+html_domainsAddDomainTitle=Aggiungi nome dominio HTML:
 html_domainsAddDomain=Nome dominio HTML:
-plaintext_domainsAddDomainTitle=Aggiungere un nome dominio testo semplice:
+plaintext_domainsAddDomainTitle=Aggiungi nome dominio testo semplice:
 plaintext_domainsAddDomain=Nome dominio testo semplice:
 
-domainNameErrorTitle=Errore nel nome di dominio
+domainNameErrorTitle=Errore nome di dominio
 #### LOCALIZATION NOTE: do not translate %S
 domainDuplicationError=Il nome di dominio %S esiste già in una lista HTML o Testo semplice.
 
 #### Junk
-confirmResetJunkTrainingTitle=Confermare
-confirmResetJunkTrainingText=Si è certi di voler azzerare i dati di autoistruzione del filtro incrementale?
+confirmResetJunkTrainingTitle=Conferma
+confirmResetJunkTrainingText=Sei sicuro di voler ripristinare i dati di training del filtro adattivo?
 
 #### Master Password
 
-setMasterPassword=Set Master Password...
-changeMasterPassword=Cambiare la master password ...
+setMasterPassword=Imposta Password principale...
+changeMasterPassword=Cambia la Password principale...
 
 #### Downloads
-desktopFolderName=Desktop
+desktopFolderName=Scrivania
```

Comment:
CORRECTIONS NEEDED. "Desktop" remains unchanged in Italian. The correct entry should be: 

+desktopFolderName=Desktop